### ISSUE TITLE
Installation of .NET 8.0+ disables mouse

### ISSUE THREAD

**Issue by D-lete** [dotnet/runtime#103343]
### Description

Installation of .NET runtimes appears to lead to mouse (touchpad and external HID) being disabled and remains un-recoverable until manual device uninstall and restart.

### Reproduction Steps

First encountered when installing DaVinci Resolve Studio, encountered again when installing .NET 8.0.0 from source. During install, mouse pointer disappeared and remained unavailable until manual uninstall and system restart.

### Expected behavior

Mouse should remain operational

### Actual behavior

Mouse was disabled

### Known Workarounds

Mouse can be recovered by manual uninstall of device and restart of system

### Configuration

.NET SDK 8.0; Windows 11 Version 10.0.22631 Build 22631; x64; encountered on Dell Latitude 7330 Rugged

**Comment by julealgon** [comment]
@D-lete did you file this issue with Dell as well?

**Comment by D-lete** [comment]
@julealgon I tried. Dell was insistent that the issue was not theirs despite my flagging that their hardware/drivers, proprietary software may have something to do with it.

**Comment by huoyaoyuan** [comment]
In this case, you may also contact your mouse manufactor to see if they have any idea.

**Comment by D-lete** [comment]
@huoyaoyuan it's an integrated trackpad on a Dell build; not sure Dell will want to hear about it 😂

**Comment by NikolaMilosavljevic** [comment]
[Triage] @D-lete it would be helpful to understand this issue in more details. Is it possible to check the Event logs for failures in Dell driver application or related software at the time of .NET 8 installation?

cc @ashnaga @jamshedd

**Comment by D-lete** [comment]
@NikolaMilosavljevic going through the event log, if I had to guess, I think the failure is loading the UMDF Driver\WUDFRd - I ran SFC and DISM but nothing relevant came of it (1 key update for WindowsSoundRecorder). 

Does the .NET affect any UMDF drivers? Is there any kind of driver verification, caching, querying or anything that goes and checks any of those kinds of files? I'm scratching my head here, and maybe there was an issue prior to .NET that simply coincided with the .NET install. When this issue was first observed it was following the install of DaVinci Resolve Studio which has a bundled .NET runtime with it. The next time the issue was observed it was after the installation of .NET 8. So it may be that they are related or again, coincidental.
